bakerydemo/static/css/main.css

```diff
@@ -804,232 +1172,325 @@ span.outline {
 }
 
 /* Homepage */
+
+/* #region -- Hero -- */
 .homepage .hero {
   margin: 0;
   padding: 200px 0 30px 0;
 }
 
-.homepage .hero h1 {
-  font-size: 2.2em;
-  text-transform: uppercase;
+.homepage .home-hero {
+  margin-bottom: 0;
+  padding-bottom: 20px;
 }
 
-.homepage .hero h1:after {
-  background-color: rgba(255,255,255, 0.7);
-  content: "";
-  display: block;
-  height: 5px;
-  margin: 20px auto;
-  width: 250px;
+@media (min-width: 766px) {
+  .homepage .home-hero {
+    padding-bottom: 140px;
+  }
 }
 
-.homepage .home-hero {
-  margin-bottom: 0;
-  padding-bottom: 60px;
-  text-align: center;
-}
 .homepage .home-hero .lead {
-  color: #ddd;
-  font-size: 1.6em;
+  font-family: "Marcellus", serif;
```

Comment:
We should consistently use our variables for this (here this uses the wrong fallback).

Reply:
Updated to use variable, interestingly this uses the correct fallback - my original definition uses the wrong one as Marcellus has serifs.